includes/Modules/AdSense.php

```diff
@@ -347,32 +347,34 @@ protected function amp_content_add_auto_ads( $content ) {
 	}
 
 	/**
-	 * Returns the mapping between available datapoints and their services.
+	 * Gets map of datapoint to definition data for each.
 	 *
-	 * @since 1.0.0
+	 * @since n.e.x.t
 	 *
-	 * @return array Associative array of $datapoint => $service_identifier pairs.
+	 * @return array Map of datapoints to their definitions.
 	 */
-	protected function get_datapoint_services() {
+	protected function get_datapoint_definitions() {
 		return array(
-			// GET / POST.
-			'connection'     => '',
-			'account-id'     => '',
-			'client-id'      => '',
-			'use-snippet'    => '',
-			'account-status' => '',
-			// GET.
-			'account-url'    => '',
-			'reports-url'    => '',
-			'notifications'  => '',
-			'tag-permission' => '',
-			'accounts'       => 'adsense',
-			'alerts'         => 'adsense',
-			'clients'        => 'adsense',
-			'urlchannels'    => 'adsense',
-			'earnings'       => 'adsense',
-			// POST.
-			'setup-complete' => '',
+			'GET:connection'      => array( 'service' => '' ),
+			'GET:account-id'      => array( 'service' => '' ),
+			'GET:client-id'       => array( 'service' => '' ),
+			'GET:use-snippet'     => array( 'service' => '' ),
+			'GET:account-status'  => array( 'service' => '' ),
+			'GET:account-url'     => array( 'service' => '' ),
+			'GET:reports-url'     => array( 'service' => '' ),
+			'GET:notifications'   => array( 'service' => '' ),
+			'GET:tag-permission'  => array( 'service' => '' ),
+			'GET:accounts'        => array( 'service' => 'adsense' ),
+			'GET:alerts'          => array( 'service' => 'adsense' ),
+			'GET:clients'         => array( 'service' => 'adsense' ),
+			'GET:urlchannels'     => array( 'service' => 'adsense' ),
+			'GET:earnings'        => array( 'service' => 'adsense' ),
+			'POST:connection'     => array( 'service' => '' ),
+			'POST:account-id'     => array( 'service' => '' ),
+			'POST:client-id'      => array( 'service' => '' ),
+			'POST:use-snippet'    => array( 'service' => '' ),
+			'POST:account-status' => array( 'service' => '' ),
+			'POST:setup-complete' => array( 'service' => '' ),
```

Comment:
Please alpha sort by datapoint name, and secondarily by method. This is how the cases are sorted in `create_data_request` and `parse_data_request` which makes it easier to navigate.

I won't comment on all `get_datapoint_definitions` methods for this but please do this for all of them.

Reply:
Updated.

Reply:
Almost, I meant sorting by datapoint name and secondarily by method to match the sorting we have in the switch cases which follows the same logic:
![image](https://user-images.githubusercontent.com/1621608/86268786-5feabe00-bbd1-11ea-9a7a-a79a0ca197fc.png)

There's nothing enforcing this, but we did have an issue to implement this for the data request methods previously so I think it makes sense to preserve the same for these as well.

Reply:
Ah... ok, updated. Should be good now.